dot/sync/chain_sync.go

```diff
@@ -632,7 +632,7 @@ func (cs *chainSync) dispatchWorker(w *worker) {
 		"start hash %s, target hash %s, "+
 		"request data %d, direction %s",
 		w.id,
-		w.startNumber, w.targetNumber,
+		*w.startNumber, *w.targetNumber,
```

Comment:
nit can this ever be nil? We should be careful that log doesn't panic if startNumber/targetNumber is nil :thinking:

Reply:
Nice catch, moving bellow an if condition that checks this case `if w.targetNumber == nil || w.startNumber == nil`